src/lint/track_config.nim

```diff
@@ -228,8 +228,12 @@ const tags = [
 ].toHashSet()
 
 proc hasValidTags(data: JsonNode; path: Path): bool =
+  const errorAnnotation = "Tracks are annotated with tags to allow searching for tracks with certain tags.\n" &
+                          "The tags should be chosen based on the general usage of their language.\n" &
+                          "For more information and the full list of supported tags see:\n" &
+                          "https://exercism.org/docs/building/tracks/config-json#h-tags"
```

Comment:
```suggestion
  const errorAnnotation = """
    Tracks are annotated with tags to allow searching for tracks with certain tags.
    The tags should be chosen based on the general usage of their language.
    For more information and the full list of supported tags see:
    https://exercism.org/docs/building/tracks/config-json#h-tags""".unindent()
```

Nit: can we use triple-quotes string literals and `unindent`? It's the style I favored elsewhere, with the rationale that it's a little cleaner.

But writing multiline strings in Nim has always felt a bit inelegant to me - I don't love either style.

See [nim-lang/RFCs#161](https://github-redirect.dependabot.com/nim-lang/RFCs/issues/161) and [nim-lang/RFCs#248](https://github-redirect.dependabot.com/nim-lang/RFCs/issues/248) (with redirects via dependabot).